proposals/lip-0043.md

```diff
@@ -152,51 +154,45 @@ where `REGISTRATION_FEE` is a constant given in the protocol.
 
 ```java
 sidechainRegParams = {
-  "type": "object",
-  "properties": {
-    "name": {
-      "dataType": "string",
-      "fieldNumber": 1
-    },
-    "genesisBlockID": {
-      "dataType":"bytes",
-      "fieldNumber": 2
-    },
-    "initValidators": {
-      "type":"object",
-      "fieldNumber": 3,
-      "required": [
-        "keys",
-        "weights",
-        "certificateThreshold"
-      ],
-      "properties": {
-        "keys": {
-          "type": "array",
-          "items": {
-            "dataType": "bytes"
-          },
-          "fieldNumber": 1
+    "type": "object",
```

Comment:
not mandatory, but since we are at it you could consider update the schemas to the new format (e.g. "required" comes after "type")

Reply:
schemas updated as said

Reply:
schemas updated as suggested